### PR TITLE
Fil 190: Added Generic Foreign Key to Placeholder model

### PR DESCRIPTION
**Pull request by anirbanlahiri-fidelity**

### CODE REVIEW

File: cms/models/placeholdermodel.py [hunk 1]
```diff
@@ -38,6 +40,9 @@ class Placeholder(models.Model):
     cache_placeholder = True
     is_static = False
     is_editable = True
+    content_type = models.ForeignKey(ContentType,  blank=True, null=True, on_delete=models.SET_NULL)
```
Comment (webbyfox) [inline cms/models/placeholdermodel.py]: Extra space after `ContentType,  `

File: cms/migrations/0022_auto_20180719_1638.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+        ('cms', '0021_auto_20180507_1432'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='placeholder',
+            name='content_type',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to='contenttypes.ContentType'),
```
Comment (Aiky30) [inline cms/migrations/0022_auto_20180719_1638.py]: This line is too long. Paulo will pick up on this.

**Review by Aiky30 — COMMENTED** [review]
Two changes to make, one from Riz and one from me. I will need to let you know the process to get this reviewed by Paulo one the changes are made.

File: cms/migrations/0022_auto_20180719_1638.py (hunk 2 above)
Comment (anirbanlahiri-fidelity) [reply cms/migrations/0022_auto_20180719_1638.py]: PEP8 recommends a line length of 79 which most organizations consider as too short and usually use 120 instead. In the Django CMS codebase as well I find that a lot of lines violate the constraint of 80 characters.  In FIL and Divio is there a defacto line length limit?

Comment (webbyfox) [reply cms/migrations/0022_auto_20180719_1638.py]: I wouldn't touch anything that generated as part of migration script.

**Comment by monikasulik** [comment]
Gone to divio review, closing.

**Pull request closed without merging**